src/frontends/tensorflow_common/src/op/reciprocal.cpp

```diff
@@ -51,6 +55,9 @@ OutputVector translate_reciprocal_op(const NodeContext& node) {
         return {reciprocal};
     }
 }
+template OutputVector translate_binary_op<v1::Add>(const NodeContext& node);
+template OutputVector translate_binary_op<v1::Divide>(const NodeContext& node);
+template OutputVector translate_binary_op<v1::Power>(const NodeContext& node);
```

Comment:
strange lines, we do not need them because they do not release to reciprocal. Please revert them